README.md

```diff
@@ -141,7 +163,19 @@ To install the requirements, follow this procedure:
   <img src="imgs/demo_forex.gif" width="50%">
 </p>
 
-## IP Address
+### Home Automation
+
+- Author: [@Jumitti](https://github.com/Jumitti)
+- Repository: [Jumitti/lcd_home_automation](https://github.com/Jumitti/lcd_home_automation)
+
+This implementation shows how to use the LCD to display messages from temperature sensors (see below) and services such as Spotify and Trakt. The implementation also features an integration with Telegram to turn the LCD backlight on/off and send a few other commands to control the host machine (e.g., get the temperature, reboot, shutdown). [@Jumitti](https://github.com/Jumitti) documented the project on its own [Home Automation repository](https://github.com/Jumitti/lcd_home_automation), so make sure to check it out if you want to learn more about it.
+
+<p align="center">
+  <img src="imgs/demo_home_automation.jpg" width="50%">
+</p>
+
```

Comment:
@Jumitti could you please take a look at this part of the doc to make sure its correct?

Reply:
Hello @cgomesu

I think your description is very good, maybe you can use it rather demo_HApart2_now_playing_music.gif
On this demo there are temperature sensing and Spotify API used.

This is only a suggestion, If you prefer to keep this one for the homogeneity of the readme.md that's fine

Reply:
thanks for double checking. my main concern was the size of the gif (the original is 19MB). I went ahead and optimized that one as well as the other ones above 3MB. the doc now uses a gif for the home automation demo, as requested.